misc/plunk-gen.js

```diff
@@ -89,4 +90,40 @@ ${generateTags(['Angular', 'Bootstrap', 'ng-bootstrap', capitalize(componentName
 </html>`;
 }
 
+function replacePlaceholders(source) {
+  const versions = getVersions();
+  var result = source;
+  Object.keys(versions).forEach((lib) => {
+    result = result.replace(new RegExp(`%${lib}%`, 'g'), versions[lib]);
+  });
+  return result;
+}
+
+function getVersions() {
+  return {
+    angularCore: getVersion(false, '@angular/core'),
+    angularCommon: getVersion(false, '@angular/common'),
```

Comment:
You're right of course. I'll rename it to angularVersion and use it everywhere we can.